test/test_podman_baseline.sh

```diff
@@ -81,6 +81,30 @@ echo $image
 ########
 podman run --rm $image ls -alF /etc
 
+########
```

Comment:
This is a NOP. This "baseline" file, whatever it is, is not run or used anywhere. (I'll look into getting rid of it). Tests exist in `test/system` and `test/e2e`.

Reply:
I've removed this change and added a coupe of tests to `test/e2e/ps_test.go`